lca-cli/seedcreator/seedcreator.go

```diff
@@ -304,12 +304,13 @@ func (s *SeedCreator) backupVar() error {
 	// Define the 'exclude' patterns
 	excludePatterns := []string{
 		"/var/tmp/*",
-		"/var/lib/log/*",
 		"/var/log/*",
+		"/var/lib/log/*",
+		"/var/lib/cni/bin/*",
 		"/var/lib/containers/*",
 		"/var/lib/kubelet/pods/*",
-		"/var/lib/cni/bin/*",
-		"/var/lib/ovn-ic/etc/ovnkube-node-certs/*",
+		common.OvnNodeCerts + "/*",
+		common.ImageRegistryAuthFile,
```

Comment:
@eranco74 @leo8a i actually think it will not be very easy just to avoid bringing file. 
File is managed by mco and we can't just override it.

Reply:
@leo8a did you try to run full flow and see what is going on with cluster after reboot?

Reply:
We also need to consider how to change the pull secret reboot-free, does MCO even allow changing it without a reboot?

Reply:
I see, thanks for commenting.

will /hold this and try full flow in a 2e2 env to see what could go wrong (if any) after the pivot.